machine/reference/inspect.md

```diff
@@ -61,8 +61,10 @@ For the most part, you can pick out any field from the JSON in a fairly
 straightforward manner.
 
 ```none
+{% raw %}
 $ docker-machine inspect --format='{{.Driver.IPAddress}}' dev
```

Comment:
@joaofnfernandes The second dot warning is not actually about the log format at all! It's about a block of invalid JSON which has a truncation mark '...' in it. Let me push some more changes into this PR to get warning-free.